model/Security/Classes/VexNotAffectedVulnAssessmentRelationship.md

```diff
@@ -48,13 +48,13 @@ for VEX.
 ## Metadata
 
 - name: VexNotAffectedVulnAssessmentRelationship
-- SubclassOf:  VexVulnAssessmentRelationship
+- SubclassOf: VexVulnAssessmentRelationship
 - Instantiability: Concrete
 
 ## Properties
 
 - justification
```

Comment:
Suggest changing this name for consistency across the security properties that use vocabularies.
```suggestion
- justificationType
```

Reply:
For consistency we should also change `Security/Classes/SsvcVulnAssessmentRelationship` Properties from `decision` to `decisionType`

Reply:
Thanks for the suggestions, @jeff-schutt, I included them in the two new commits.

Reply:
LGTM